pkg/monitoring/domainstats/prometheus/prometheus_test.go

```diff
@@ -99,7 +103,7 @@ var _ = Describe("Prometheus", func() {
 				},
 			}
 			vmi := k6tv1.VirtualMachineInstance{}
-			ps.Report("test", &vmi, vmStats)
+			ps.Report("test", &vmi, vmStats, emptyVmFSStats)
```

Comment:
@machadovilaca Here you've left the `vmStats` although you've refactored its name in previous places

Reply:
changed when using the suggested struct